be/src/exec/iceberg/iceberg_delete_builder.cpp

```diff
@@ -175,6 +180,66 @@ Status ORCEqualityDeleteBuilder::build(const std::string& timezone, const std::s
     return Status::OK();
 }
 
+Status ParquetEqualityDeleteBuilder::build(const std::string& timezone, const std::string& delete_file_path,
+                                           int64_t file_length,
+                                           const std::shared_ptr<DefaultMORProcessor> mor_processor,
+                                           std::vector<SlotDescriptor*> slot_descs,
+                                           TupleDescriptor* delete_column_tuple_desc,
+                                           const TIcebergSchema* iceberg_equal_delete_schema,
+                                           RuntimeState* state) {
+    std::unique_ptr<RandomAccessFile> file;
+    ASSIGN_OR_RETURN(file, _fs->new_random_access_file(delete_file_path));
```

Comment:
If the file stored in S3/Hdfs, is it really a random access file or well-implemented random access file?  if not so, does there exist performance bottleneck?